public/Buttons/ShaikhHeeba07/style.css

```diff
@@ -0,0 +1,126 @@
+.btn-62,
+.btn-62 *,
+.btn-62 :after,
+.btn-62 :before,
+.btn-62:after,
+.btn-62:before {
+  border: 0 solid;
+  box-sizing: border-box;
+}
+body{
+   background-color: #CCFFBD;
```

Comment:
add this here.
display:flex;
justify-content:center;
align-items:center;

Reply:
Do I have to add this in the body ?